test/connectivity.jl

```diff
@@ -75,31 +77,84 @@
       scc_as_subgraphs = map(i -> graph[i], scc)
       return all(is_strongly_connected, scc_as_subgraphs)
     end
+    
+    function scc_k_ok(graph)
+      """Check that all SCC really are strongly connected"""
```

Comment:
this should be a comment, not a docstring (looks sort of like python :) )

Reply:
Changed.